src/components/PropertyDetailSection.tsx

```diff
@@ -166,16 +164,7 @@ const PropertyDetailSection: FC<PropertyDetailSectionProps> = ({
     return featuresInView.slice(start, end);
   }, [page, featuresInView, smallScreenMode]);
 
-  return hasLoadingError ? (
```

Comment:
The error UI was removed because changing the map style temporarily causes an error due to no items being in the property list. I noticed there is a separate UI designed specifically for "no results," rather than displaying the error UI. I wanted to bring this to attention.

Reply:
@DonovanAndrews300 before removing the error UI, can you look at PR #732? The error UI appears if the map fails to fetch data. Removing that UI might have unexpected consequences and re-introduce a bug that was fixed by that previous PR.

Reply:
Thanks for this you were absolutely correct in removing the error UI causing breaking changes for the reasons mentioned in the PR you linked. I added the UI back in and instead just had it skip setting the error state in the map component depending on the error received.